packages/web/src/components/social-button/SocialButton.module.css

```diff
@@ -0,0 +1,20 @@
+.socialButton {
+  width: 100%;
+  margin-bottom: 8px;
+  height: 64px !important;
+}
+
+.socialButton svg {
+  height: 20px !important;
+  width: 20px !important;
+}
+
+.socialButton .icon path {
+  fill: var(--static-white);
+}
+
+.text {
+  font-size: var(--font-xl) !important;
```

Comment:
one day we can address non needing !important :/ i guess it has to do with our button style specificity?

Reply:
Yeah it was either that or adding extra classes to out-specify the styles that are on the button :(

css in js wen